app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] Kindly note that as you have overwritten the default foreign_key `user_id` in all associations with the user, you need also to specify the foreign_key(`author_id`)  here in these associations above :point_up: . Hint :point_down: 
```rb 
class User < ApplicationRecord
  has_many :posts, foreign_key: :author_id
  has_many :comments, foreign_key: :author_id
  has_many :likes, foreign_key: :author_id
end

```